README.npm.md

```diff
@@ -0,0 +1,217 @@
+
+## Features
+
+- ✨ Complete visualisation of a Kedro project and its pipelines
+- 🎨 Supports light & dark themes out of the box
+- 🚀 Scales to big pipelines with hundreds of nodes
+- 🔎 Highly interactive, filterable and searchable
+- 🔬 Focus mode for modular pipeline visualisation
+- 📊 Rich metadata side panel to display parameters, plots, and more.
+- 📊 Supports all types of [Plotly charts](https://plotly.com/javascript/)
+- ♻️ Autoreload on code change
+- 🧪 Supports tracking and comparing runs in a Kedro project
+- 🎩 Many more to come
+
+## Installation
+
+- As a standalone React component (for embedding Kedro-Viz in your web application).
+
+  To install the standalone React component:
+
+  ```bash
+  npm install @quantumblack/kedro-viz
+  ```
+
+## Usage
+
+To use Kedro-Viz as a standalone React component, you can follow the example below. Please note that Kedro-Viz does not support server-side rendering (SSR). If you are using Next.js or another SSR framework, you should be aware of this limitation.
+
+```javascript
+import KedroViz from '@quantumblack/kedro-viz';
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+
+const MyApp = () => (
+  <div style={{height: `100vh`}}>
+    <KedroViz
+      data={json}
+      options={/* Options to configure Kedro Viz */}
+    />
+  </div>
+); 
+```
+
+To use with NextJS:
+
+```javascript
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+import dynamic from 'next/dynamic';
+
+const NoSSRKedro = dynamic(() => import('@quantumblack/kedro-viz'), {
+  ssr: false,
+});
+
+const MyApp = () => <NoSSRKedro data={json} />;
+```
+
+The JSON can be obtained by running:
+
+```bash
+kedro viz run --save-file=<path-to-save-kedro-viz-data-to-a-directory>
+```
+
+On successful execution of the command above, it will generate a folder at the specified directory, containing the following structure: 
+
+```
+<filename>/api/
+├── main
+├── nodes
+│   ├── 23c94afb
+│   ├── 28754fab
+│   ├── 2ab3579f
+│   ├── 329e963c
+│   ├── 369acf98
+│   └── ...
+└── pipelines
+    ├── __default__
+    ├── data_processing
+    ├── data_science
+    └── ...
+```
+
+Use the `main` file as the input JSON for the `data` prop in your Kedro-Viz component.
+
+We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.
+
+**_Our documentation contains [additional examples on how to visualise with Kedro-Viz.](https://docs.kedro.org/en/stable/visualisation/index.html)_**
+
+## Configure Kedro-viz with `options`
+
+The example below demonstrates how to configure your kedro-viz using different `options`. 
+
+```
+<KedroViz
+    data={json}
+    options={
+      expandAllPipelines: false,
+      display: {
```

Comment:
[Nit] If we can arrange these in alphabetical order it would be nice in the docs. Same goes for the table below under each section if we can have an order.